rnndb/graph/g80_3d.xml

```diff
@@ -507,7 +503,7 @@ xsi:schemaLocation="http://nouveau.freedesktop.org/ rules-ng.xsd">
 		</reg32>
 		<reg32 offset="0x1240" name="RT_HORIZ" length="8" stride="8">
 			<bitfield low="0" high="27" name="WIDTH"/>
-			<bitfield pos="31" name="LINEAR">
+			<bitfield pos="31" name="FORMAT" type="inv_memory_format">
```

Comment:
It's going to be hard to remember which is memory_format and which is inv_memory_format..